pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -144,6 +144,18 @@ let
         };
       };
 
+      adzero.vscode-sievehighlight = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-sievehighlight";
+          publisher = "adzero";
+          version = "1.0.6";
+          hash = "sha256-8Ompv792eI2kIH+5+KPL9jAf88xsMGQewHEQwi8BhoQ=";
+        };
+        meta = {
```

Comment:
Meta fields are missing.

Reply:
Sorry must have got lost when stashing to nixfmt etc. readded it again.